.agent/skills/ss14-ecs-systems/SKILL.md

```
@@ -0,0 +1,451 @@
+    {
+        base.Initialize();
+        // Клиентские подписки: UI, визуалы, звуки
+    }
+}
+```
+
+## Partial-класс декомпозиция
+
+Для сложных систем логика разбивается на несколько файлов через `partial class`. Каждый файл отвечает за свою подсистему:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)